config/default/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-cluster-api/cluster-api-controller:v0.1.4
+      - image: k8s.gcr.io/cluster-api/cluster-api-controller:v0.1.5
```

Comment:
This *should* be the home of the promoted image after running through the Image Promotion process.